ait/core/server/handler.py

```diff
@@ -125,18 +123,24 @@ def handle(self, input_data):
         Returns:
             tuple of packet UID and packet data field
         """
+
         packet = bytearray(input_data)
+        if (len(packet) < 13):
+            ait.core.log.info('CCSDSPacketHandler: Insufficient packet length.')
+            return
         packet_apid = int(binascii.hexlify(packet[6:8]), 16) & 0x07FF
         if packet_apid not in self.packet_types:
             msg = 'CCSDSPacketHandler: Packet APID {} not present in config.'.format(packet_apid)
-            msg += ' Available packet APIDs are {}'.format(self.packet_types.keys())
             ait.core.log.info(msg)
             return
         packet_name = self.packet_types[packet_apid]
         tlm_dict = tlm.getDefaultDict()
         packet_uid = tlm_dict[packet_name].uid
 
         packet_data_length = int(binascii.hexlify(packet[10:12]), 16) + 1
+        if (len(packet) < 12 + packet_data_length):
+            ait.core.log.info('CCSDSPacketHandler: Insufficient packet length.')
```

Comment:
Same as above. "Insufficient packet length" isn't wrong but more details would be better!

Reply:
Also its confusing that this is the exact same error message as above - that won't be helpful to users trying to identify what the exact problem is.